package.json

```diff
@@ -58,6 +58,7 @@
     "react-redux": "^6.0.1",
     "react-router-dom": "^5.0.0",
     "react-share": "^2.4.0",
+    "react-tiny-fab": "^2.0.0",
```

Comment:
@olajide1234 what are we using this for?

Reply:
For the floating action button.